### PR TITLE
FOGL-343 base work is for config manager omf translator integration is done

### PR DESCRIPTION
**Pull request by ashwinscale**
Stefano will complete the work and put the rest of the configuration in

### CODE REVIEW

File: src/python/foglamp/translators/omf_translator.py [hunk 1]
```diff
@@ -143,6 +142,17 @@
 """Associates the asset code to the corresponding type"""
 
 
+_DEFAULT_PURGE_CONFIG = {
```
Comment (praveen-garg) [inline src/python/foglamp/translators/omf_translator.py]: _DEFAULT_OMF_TRANSLATOR_CONFIG

Though as per Shakespeare , 
> “What's in a name? that which we call a rose
> By any other name would smell as sweet.”

File: src/python/foglamp/translators/omf_translator.py [hunk 2]
```diff
@@ -143,6 +142,17 @@
...
+    "relay_server_name": {
+        "description": "Host name/IP of OMF connector relay",
+        "type": "string",
+        "default": "WIN-4M7ODKB0RH2"
+    }
+}
+_CONFIG_CATEGORY_NAME = 'OMF_TRANS'
```
Comment (praveen-garg) [inline src/python/foglamp/translators/omf_translator.py]: may be trivial, but let's try to keep name self explanatory? `OMF_TRANSLATOR`

File: src/python/foglamp/translators/omf_translator.py [hunk 3]
```diff
@@ -959,11 +967,17 @@ def omf_object_creation():
     start_message = "\n" + _module_name + " - Ver " + version + "" + prg_text + "\n" + __copyright__ + "\n"
     debug_msg_write("", "{0}".format(start_message))
     debug_msg_write("INFO", _message_list["i000002"])
+    event_loop = asyncio.get_event_loop()
+    event_loop.run_until_complete(configuration_manager.create_category(_CONFIG_CATEGORY_NAME, _DEFAULT_PURGE_CONFIG,
```
Comment (praveen-garg) [inline src/python/foglamp/translators/omf_translator.py]: as above, the name!

File: src/python/foglamp/translators/omf_translator.py (hunk 2 above)
Comment (ashwinscale) [inline src/python/foglamp/translators/omf_translator.py]: 10 character max on ddl

File: src/python/foglamp/translators/omf_translator.py (hunk 1 above)
Comment (ashwinscale) [inline src/python/foglamp/translators/omf_translator.py]: done

File: src/python/foglamp/translators/omf_translator.py (hunk 3 above)
Comment (ashwinscale) [inline src/python/foglamp/translators/omf_translator.py]: done

File: src/python/foglamp/translators/omf_translator.py (hunk 2 above)
Comment (praveen-garg) [inline src/python/foglamp/translators/omf_translator.py]: ah! Yes I slipped on that.

**Pull request merged**